templates/update.html

```diff
@@ -0,0 +1,23 @@
+{% extends "layout.html" %}
```

Comment:
This file needs to be in the todo_app/templates folder rather than just /templates